doc/python/2d-histogram-contour.md

```diff
@@ -32,6 +32,19 @@ jupyter:
     permalink: python/2d-histogram-contour/
     thumbnail: thumbnail/hist2dcontour.png
 ---
+#### Contour Plot in Plotly Express
+
+[Plotly Express](/python/plotly-express/) is the easy-to-use, high-level interface to Plotly, which [operates on "tidy" data](/python/px-arguments/).
```

Comment:
I think we should say "plotly.py" instead of "Plotly" here.

Reply:
Let's do this another day... this is the standard sentence we have in every doc page now :)